tssc/step_implementers/validate_environment_configuration/__init__.py

```diff
@@ -0,0 +1,29 @@
+"""tssc.StepImplementers for the 'validate-environment-configuration' TSSC step.
+
+
+Step Configuration
+------------------
+All tssc.StepImplementers for this step should
+accept minimally the following configuration options.
+
+| Parameter       | Description
+|-----------------|------------
+| `yaml_path`     | Path to the files to be reviewed.
+
+Results
+-------
+All tssc.StepImplementers for this step should
+minimally produce the following step results.
+
+| Result Key       | Description
+|------------------|------------
+| TBD              |
```

Comment:
Common result keys can be added here

Reply:
Updated code to return a file for report-artifacts.